Description: Work around a new bug (#418067) in apache2.2 by explicitly creating a
 new process group in startup.pl.  Should stop 'apache -X' exit from
 SIGTERM'ing whatever the process group happens to be.
 Really closes: #416016.
Author: Angus Lees <gus@debian.org>

--- a/test/conf/startup.pl
+++ b/test/conf/startup.pl
@@ -60,6 +60,13 @@
 
     }
 
+# Bug#418067: apache2.2-mpm (at least) doesn't create a new process
+# group with apache -X.  When exiting though, it SIGTERM's whatever
+# the process group happens to be, wreaking havoc.  More importantly
+# it causes 'make test' to exit with failure.  Hack around that here:
+use POSIX ();
+POSIX::setpgid(0,0);
+
 use Embperl ;
 use Embperl::Object ;
 
